db-connector/src/main/resources/uk/nhs/adaptors/connector/dao/PatientAttachmentLogDao/insert-dummy-data.sql

```diff
@@ -0,0 +1,84 @@
+INSERT INTO patient_attachment_log(
```

Comment:
Can re rename this file to. attachement-logs-test-data? It's good to have but it's not code critical. Maybe place it in a local folder called local-test-data?